sass/index.scss

```diff
@@ -229,7 +229,60 @@ section {
 
 // Hero
 #hero {
+    display: flex;
+    flex-direction: column;
+
+    #hero-video{
+        width: 100%;
+        display: flex;
+        justify-content: flex-end;
+        z-index: -1;
+        margin-top: -200px;
+
+        video {
+            max-width: 100%;
+        }
+    }
 
+    #hero-content {
+
+        p {
+            margin-top: 2rem;
+        }
+
+        .field-row-stacked {
+            border: 2px solid $blue;
+            border-radius: 5px;
+            font-size: var(--body-font-size);
+            display: inline-block;
+            background-color: white;
+
+            input {
+                border: none;
+                outline: none;
```

Comment:
This seems to hide focus; it's hard to tell that I'm actually clicked into the field aside from the small cursor. Normally there'd be a small indication -- any chance this can be tweaked slightly?

Reply:
Also signup form, irrelevant because being replaced